shared/src/packages/surveys/libs/validation-schemas/create-survey.validation-schema.ts

```diff
@@ -0,0 +1,29 @@
+import joi from 'joi';
+
+import {
+  SurveyValidationMessage,
+  SurveyValidationRule,
+} from '../enums/enums.js';
+import { type SurveyRequestDto } from '../types/types.js';
+
+const CreateSurveyValidationSchema = joi.object<SurveyRequestDto, true>({
+  userId: joi.number().integer().required().messages({
+    'any.required': SurveyValidationMessage.USER_ID_REQUIRED,
+    'string.empty': SurveyValidationMessage.USER_ID_REQUIRED,
```

Comment:
Add a rule for number.base to check if it is a number or not

Reply:
Also I'm not sure about the use of string.empty, because we are validating for a number, not a string

Reply:
yes, you are right, I was thinking it's a string